repository/src/main/resources/alfresco/events2-context.xml

```diff
@@ -38,7 +38,6 @@
         <property name="dictionaryService" ref="dictionaryService"/>
         <property name="descriptorService" ref="descriptorComponent"/>
         <property name="eventFilterRegistry" ref="event2FilterRegistry"/>
-        <property name="event2MessageProducer" ref="event2MessageProducer"/>
```

Comment:
See my comment in the enterprise PR. Please add `eventGeneratorQueue` here to avoid the need to set it in the enterprise.